shared/profile/wot-author/index.tsx

```diff
@@ -331,16 +369,21 @@ const OtherInput = (props: {otherText: string; setOtherText: (_: string) => void
 )
 
 // Store checkedness for a list of checkboxes.
-function useCheckboxesState<T>(items: T[]): (T & {checked: boolean; onCheck: (_: boolean) => void})[] {
-  const [stateStored, setState] = React.useState<boolean[]>(items.map(() => false))
+function useCheckboxesState<T>(items: Array<T>, onCheck: () => void): [Array<T & Checkboxed>, () => void] {
+  const [stateStored, setState] = React.useState<Array<boolean>>(items.map(() => false))
   const state = items.length === stateStored.length ? stateStored : items.map(() => false)
-  return items.map((item, i) => ({
-    ...item,
-    checked: state[i],
-    onCheck: checked => {
-      setState(state.map((wasChecked, j) => (i === j ? checked : wasChecked)))
-    },
-  }))
+  const clear = () => setState(items.map(() => false))
+  return [
+    items.map((item, i) => ({
+      ...item,
+      checked: state[i],
+      onCheck: checked => {
+        setState(state.map((wasChecked, j) => (i === j ? checked : wasChecked)))
+        onCheck()
+      },
+    })),
+    clear,
+  ]
```

Comment:
Where ever you do `items.map(() => false)` you can optionally replace with

```
const initialState = Array(items.length).fill(false)
// ....
// now reset using `inititalState`
```

Reply:
Changed the other direction last time
https://github.com/keybase/client/pull/23458#discussion_r402561391